src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -127,119 +126,235 @@ private function getPath(string $resourceShortName, array $operation, bool $coll
      *
      * @return \ArrayObject
      */
-    private function getPathOperation(string $operationName, array $operation, string $method, bool $collection, ResourceMetadata $resourceMetadata, array $mimeTypes) : \ArrayObject
+    private function getPathOperation(string $operationName, array $operation, string $method, bool $collection, string $resourceClass, ResourceMetadata $resourceMetadata, array $mimeTypes, \ArrayObject $definitions) : \ArrayObject
```

Comment:
@teohhanhui why? It's an object not an array, so it's by definition a reference and it's ok to mutate it.
I don't get the benefit of using `\ArrayObject` instead of a raw array if we cannot use such features :)